.github/workflows/docs-publish.yaml

```diff
@@ -9,38 +9,54 @@ on:
   schedule:
```

Comment:
I wonder if shouldn't we remove this schedule.

Reply:
Good call, I will remove it!